package.json

```diff
@@ -18,11 +19,11 @@
     "compile": "run-s clean:lib clean:modules build:types babel:lib babel:modules",
     "prepublishOnly": "run-s compile deploy",
     "postreleasy": "vtex publish --public --verbose",
-    "lint": "eslint react --ext js,jsx,ts,tsx",
     "lint-fix": "eslint react --ext js,jsx,ts,tsx --fix --max-warnings 0",
     "prettier-conflict-check": "eslint --print-config .eslintrc | eslint-config-prettier-check",
     "postinstall": "yarn --cwd react --ignore-scripts"
   },
+  "version": "9.104.5",
```

Comment:
why? [in line 3](https://github.com/vtex/styleguide/pull/993/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3) that already exists

Reply:
Might be a rebase mistake, let me check.